src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import cn from 'classnames';
+import { useParams, Link } from 'react-router-dom';
+import { Person } from '../../types/Person';
+
+interface Props {
+  person: Person,
+  findPerson: (name: string | null) => Person | undefined,
+}
+
+export const PersonItem: React.FC<Props> = ({
+  person,
+  findPerson,
+}) => {
+  const {
+    name,
+    sex,
+    born,
+    died,
+    motherName,
+    fatherName,
+    slug,
+  } = person;
+
+  const { personSlug } = useParams();
+
+  const mother = findPerson(motherName);
+
+  const father = findPerson(fatherName);
```

Comment:
find the mother and father on initial load to not call `findPerson` function on each rerender